lib/ui/src/components/notifications/item.stories.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-
 import NotificationItem from './item';
 
```

Comment:
Gentle request: Can we add an action here so that we can confirm onClick in the Actions addon tab?

If you use [Args](https://storybook.js.org/docs/react/writing-stories/args) we should get this automatically.